models/candidate.js

```diff
@@ -7,21 +7,33 @@ const mongoose = require("mongoose"),
             type: String,
             required: true
         },
-        start_date: {
-            type: Date,
-            default: Date.now
-        },
+        job_type : String,
         expected_salary: String,
         current_location: String,
         preferred_location: [String],
-        willing_to_relocate: Boolean,
         work_experience: [String],
-        hard_skills: [String],
-        soft_skills: [String],
+        hard_skills: [{
+          name: String,
+          importance: {
+            type: Number,
+            min : 1,
+            max : 4,
+            required : true
+          }
+        }],
+        soft_skills: [{
+          type : String,
+          required : true
+        }],
         other_aspects: [String],
         work_culture_preferences: [{
-            type: String,
-            required: true
+          name: String,
+          importance: {
```

Comment:
Did we really want to have an importance for work_culture_preferences? I thought we agreed on only having importance for hard skills
Might be mistaken here though

Reply:
We did want the importance of work culture in candidate.js but not in job_offer bc the work culture is what the recruiter provides